app/geodiffutils.cpp

```diff
@@ -0,0 +1,96 @@
+#include "geodiffutils.h"
```

Comment:
missing copyright header

Reply:
hmm there are more files without the header - let's address that separately